Documentation/getting-started.md

```diff
@@ -39,3 +39,21 @@ Sample data is available for download from the [SlicerAutoscoperM Sample Data](/
 * Ankle BVR data - This data was provided by Michael Rainbow.
   * Three frames of movement are included in the sample data.
   * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.
+
+## Glossary
+
+This section defines terms that are commonly used by Autoscoper and SlicerAutoscoperM
+
+* **3D Computed Tomography (CT)**: Computerized x-ray imaging method producing volumetric density data of a region of interest. Saved in DICOM format.
+* **4D CT**: Temporal 3D CT
+* **Biplaner Videoradiography (BVR)**: Two calibrated x-ray sources and high-speed video camera sets that allow for the synchronous collection of radiographs from two perspectives. Camera models from the calibrations allow for unified 3D World construction from perspective 2D views.
+* **Rigid Body**: Commonly a bone or an implant. The rigid body is the region of interest within a 3DCT. Users segment or isolate the data which represents the rigid body. Often the rigid body surface file is exported as a bone mesh model (STL). Autoscoper is used to track the kinematics (rotation and translation) of the rigid body in 3-dimensional space.
+* **Partial Volume (PVOL)**: Saved as a TIFF stack, the PVOL is a volumetric image data file. A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body and cropped to its dimensions.
+* **AUT Model (AUT)**: A representation of the rigid body STL as parsed in the Autoscoper 3D World
+* **Digitally Reconstructed Radiograph (DRR)**: When a version of the PVOL is loaded into Autoscoper, the perspective camera models cast the volumetric density data onto the same image plane as the radiographs to generate a synthetic radiograph
```

Comment:
```suggestion
* **Digitally Reconstructed Radiograph (DRR)**: A synthetic radiograph generated when a PVOL is loaded into Autoscoper. Perspective camera models cast the volumetric density data onto the same image plane as the radiographs, simulating the radiograph digitally.
```

I removed the words `a version of the` associated with the PVOL. If we can clarify, this can be added back.